### DISCUSSION TITLE
Maximum repo size

### DISCUSSION THREAD

**Discussion by viethung0823** [Vinzent03/obsidian-git#650] [General]
I found that GitHub limit each repo has maximum size is 100GB. At that time we won't able to push anything. Have you thought about this? What is the solution for us?

**Comment by Vinzent03** [comment]
I think 100gb is already very huge for a git repository. I guess the most is attachments, which I think should be stored in git-lfs at that size. I think git-lfs is not included in the 100gb.
But this plugin works with any git server. So you might find another host.
